stdlib/PCRE2_jll/src/PCRE2_jll.jl

```diff
@@ -5,41 +5,18 @@ baremodule PCRE2_jll
 using Base, Libdl
 Base.Experimental.@compiler_options compile=min optimize=0 infer=false
 
-const PATH_list = String[]
-const LIBPATH_list = String[]
 
 export libpcre2_8
 
-# These get calculated in __init__()
-const PATH = Ref("")
-const LIBPATH = Ref("")
-artifact_dir::String = ""
-libpcre2_8_handle::Ptr{Cvoid} = C_NULL
-libpcre2_8_path::String = ""
-
 if Sys.iswindows()
-    const libpcre2_8 = "libpcre2-8-0.dll"
+    const libpcre2_8_name = "bin/libpcre2-8-0.dll"
```

Comment:
OK I guess this is where the `libpcre_2_8` bit comes from? Ah well, if that simplifies staying in sync with the "regular" JLLs, it makes some sense